lib/queuePopulator/LogReader.js

```diff
@@ -355,14 +369,7 @@ class LogReader {
                     info: logRes.info,
                 });
             logStats.nbLogRecordsRead += 1;
-
-            this._setEntryBatch(entriesToPublish);
-            record.entries.forEach(entry => {
-                logStats.nbLogEntriesRead += 1;
-
-                this._processLogEntry(batchState, record, entry);
-            });
-            this._unsetEntryBatch();
+            batchState.currentRecord = record;
```

Comment:
`logRes.log` is a stream of record objects, so you may have more than one record (usually the configured batch size like 1000 at a time). If you set it here it will only keep the last record for `processFilterEntries` so the 999 previous ones will not be processed.